huaweicloud/services/cbr/resource_huaweicloud_cbr_vault.go

```diff
@@ -38,15 +43,15 @@ var ResourceType map[string]string = map[string]string{
 	VaultTypeTurbo:  ResourceTypeTurbo,
 }
 
-func ResourceCBRVaultV3() *schema.Resource {
+func ResourceVault() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceCBRVaultV3Create,
-		Read:   resourceCBRVaultV3Read,
-		Update: resourceCBRVaultV3Update,
-		Delete: resourceCBRVaultV3Delete,
+		CreateContext: resourceVaultCreate,
+		ReadContext:   resourceVaultRead,
+		UpdateContext: resourceVaultUpdate,
+		DeleteContext: resourceVaultDelete,
 
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			StateContext: schema.ImportStatePassthroughContext,
 		},
 
 		Schema: map[string]*schema.Schema{
```

Comment:
please add a timeout block because we need to wait the order

Reply:
sure, I has been finished.